app/user.go

```diff
@@ -618,11 +618,11 @@ func (a *App) GetTeamGroupUsers(teamID string) ([]*model.User, *model.AppError)
 
 // GetChannelGroupUsers returns the users who are associated to the channel via GroupChannels and GroupMembers.
 func (a *App) GetChannelGroupUsers(channelID string) ([]*model.User, *model.AppError) {
-	result := <-a.Srv.Store.User().GetChannelGroupUsers(channelID)
```

Comment:
Here you can just return `a.Srv.Store.User().GetChannelGroupUsers(channelID)`

Reply:
Yeah, that makes sense. I'll update. Thanks!